enterprise/configuration.mdx

```diff
@@ -0,0 +1,53 @@
+---
```

Comment:
Let's use Enterprise Edition (or EE shorthand) across the board. We want to be consistent for this one where possible